plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3909,7 +3909,7 @@ public void testGetIcebergTableProperties()
     {
         assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT)");
         assertThat(query("SELECT * FROM \"test_iceberg_get_table_props$properties\""))
-                .matches(format("VALUES (VARCHAR 'write.format.default', VARCHAR '%s')", format.name()));
+                .containsAll(format("VALUES (VARCHAR 'write.format.default', VARCHAR '%s')", format.name()));
```

Comment:
It would be better to extract a method for the expected table properties and override in REST test.